package/MDAnalysis/analysis/contacts.py

```diff
@@ -396,11 +396,12 @@ def __init__(self, u, selection, refgroup, method="hard_cut", radius=4.5,
             dictionary of additional kwargs passed to `method`. Check
             respective functions for reasonable values.
         start : int, optional
-            First frame of trajectory to analyse, Default: 0
+            First frame of trajectory to analyse, Default: None becomes 0.
         stop : int, optional
-            Last frame of trajectory to analyse, Default: -1
+            Frame index to stop analysis. Default: None becomes
+            n_frames. Iteration stops *before* this frame number.
```

Comment:
On 27/07/16 20:25, Oliver Beckstein wrote:
>
> In package/MDAnalysis/analysis/contacts.py 
> <https://github.com/MDAnalysis/mdanalysis/pull/916#discussion_r72495216>:
>
> >          stop : int, optional
> > -            Last frame of trajectory to analyse, Default: -1
> > +            Frame index to stop analysis. Default: None becomes
> > +            n_frames. Iteration stops *before* this frame number.
>
> , which means that the trajectory would be read until the end (because 
> the first frame has index 0 and the last frame has index n_frames - 1.
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/MDAnalysis/mdanalysis/pull/916/files/79b1582267ece8ea14f27fd4bed37ef187d95544#r72495216>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/ABUWuuLApCu8e94JQMbJNZsA4rp733hgks5qZ6K0gaJpZM4JVv0L>.
>
This is an odd behavior. The default should not be -1, it should be 
None. Why would we want to stop before the end of the trajectory?